apps/tsilhqotin-language-hub/src/Components/Widgets/Footer/Footer.css

```diff
@@ -0,0 +1,105 @@
+.footer {
```

Comment:
remove footer.module.css
Please always remove empty css files if you're not going to use them.